### ISSUE TITLE
HVAC library empty in OpenStudioApp 2.0.0

### ISSUE THREAD

**Issue by jmarrec** [NREL/OpenStudio#2432]
System info:

* Sketchup 2016
* macOS Sierra 10.12.2
* OpenStudio 2.0.0 (47ba3cbe19)

----

On the Thermal Zones tab, I can see stuff in the dropdowns as usual. 

![screen shot 2017-01-09 at 13 31 10](https://cloud.githubusercontent.com/assets/5479063/21766650/03d91e76-d670-11e6-9c49-3bf6d615d635.png)

Problem is, on the `HVAC Systems` tab I cannot see anything in any of the dropdowns:

![screen shot 2017-01-09 at 13 31 31](https://cloud.githubusercontent.com/assets/5479063/21766658/0d4b5244-d670-11e6-8d95-8c9b0f738800.png)

The hvac_library.osm that's under `/usr/local/openstudio-2.0.0/OpenStudioApp.app/Contents/Resources/hvaclibrary/hvac_library.osm` does have the stuff it's supposed to have, e.g. for example in the ATUSingleDuctUncontrolled:

In an interactive prompt, loading the openstudio 2.0.0 bindings:

```ruby
[1] hvaclibrary(main)> model = osload('hvac_library.osm')
[2] hvaclibrary(main)> model.getAirTerminalSingleDuctUncontrolleds
=> [#<OpenStudio::Model::AirTerminalSingleDuctUncontrolled:0x007fde2badaca8
  @__swigtype__="_p_openstudio__model__AirTerminalSingleDuctUncontrolled">]
[3] hvaclibrary(main)> puts model.getAirTerminalSingleDuctUncontrolleds
OS:AirTerminal:SingleDuct:Uncontrolled,
  {b883cca7-f436-48ce-ad23-7e8310c7a85d}, ! Handle
  Diffuser,                               ! Name
  {9f54092d-a4a8-41b8-a381-c4c332ecb843}, ! Availability Schedule Name
  ,                                       ! Inlet Air Node Name
  ,                                       ! Zone Supply Air Node Name
  AutoSize;                               ! Maximum Air Flow Rate {m3/s}

```

**Comment by jmarrec** [comment]
I have no explanation for this, but after reloading the model, I no longer have this problem...

**Comment by kbenne** [comment]
I guess keep an eye on this and report back if you can reproduce it.

**Comment by DavidGoldwasser** [comment]
Users reporting this on windows machines when launch OSM by double clicking it, but not when loading through file menu.
https://github.com/NREL/OpenStudio/issues/2432

I could see it on windows but not mac, I'll try again on mac with home machine.

**Comment by aaron-boranian** [comment]
@DavidGoldwasser, @kbenne, @jmarrec for what it's worth, I've seen this in OS v2.1 as well during a workshop last week. One solution we found was to make a minor change, then use the menu command **File / Revert to Saved** (see below). When OpenStudio reopens, the HVAC library components appear like they normally should.
![screen shot 2017-05-15 at 10 17 34 am 2](https://cloud.githubusercontent.com/assets/15160917/26067651/19aeb62a-3958-11e7-8870-f4cd0ff6bce5.png)

**Comment by jmarrec** [comment]
Had the same thing happen to me in 2.1.1 on Mac today as well. File > Loa library > hvac_library.osm did the trick.

Envoyé de mon iPhone

**Comment by jmarrec** [comment]
Can someone test it on mac with 2.6.0 RC? @DavidGoldwasser for eg?
A windows user too perhaps @macumber?

On Ubuntu I'm hit by #3158

**Comment by DavidGoldwasser** [comment]
@jmarrec Below are the only categories that didn't have anything under them on mac
- Air Terminal Four Pipe Beam
- Plant Component User Defined
- Setpoint Manager Humidity Minimum

**Comment by jmarrec** [comment]
@DavidGoldwasser I guess that doesn't related to the original title, but should still be fixed...

Though it might be already ok now, since with current develop (9edf6e6a7845053047b4dfc82113e08cade23fc9, 2.6.0 basically) I do have stuff under these three on Ubuntu.

**Comment by jmarrec** [comment]
Outdated issue, closing unless someone runs into it. It was perhaps related to #3190 fixed in #3191
